style_guide/style_guide.md

```diff
@@ -67,6 +67,16 @@ Additionally, developers of this project should be mindful of application securi
 - Each function/method should have at minimum a description, `Parameters` (if necessary), and `Returns` (if necessary) sections
 
 
+`jwql`-Specific Variable Value/Type Standards
+---------------------------------------------
+
+To the extent possible, `jwql` shall define frequently-used variable types/values consistently. A list of adopted standards is provided below:
+
+- **JWST instrument names**: In all internal references and structures (e.g. dictionaries) instrument names shall be all lower-case strings, i.e. one of `fgs`, `miri`, `niriss`, `nircam`, `nirspec`. When variations are required for interfaces, e.g. `Nircam` for MAST, `NIRCam` or `NIRCAM` for SIAF, etc. these should be defined as dictionaries in [`jwql/utils/utils.py`](https://github.com/spacetelescope/jwql/blob/master/jwql/utils/utils.py) and imported from there.
+
+- **Program/proposal identifiers**: JWST program IDs shall be stored and referred to internally as integers and parsed to strings only when needed. For example, the inputs `"001144"` and `"1144"` shall both be converted to an integer variable with value `1144`.  
```

Comment:
@Johannes-Sahlmann I wonder if there is any value in defaulting to storing program identifiers as strings, instead of integers.  Though program IDs are comprised of integers, they are not meant to be used as numbers to be added, subtracted, etc.  I'm struggling to think of scenarios in which it would be useful to have one of these identifiers in code as a `int` class.

Another aspect that I think may be important for consistency sake is ensuring the program identifiers use all 6 digits with zero-padding when necessary (i.e. `001144` instead of `1144` or `01144`), which would be easier to enforce with strings.

Do you have any thoughts about this?

Reply:
@Johannes-Sahlmann Those are good points.  In that case, I think what you have here in the `style_guide` looks good.  In the meantime, let's try to strive for allowing functionality for both zero-padded and non-zero-padded program IDs when possible; I assume this is an easy fix for our web app search bar.